ui/static/js/confirm_handler.js

```diff
@@ -32,7 +32,11 @@ class ConfirmHandler {
             questionElement.remove();
             containerElement.appendChild(loadingElement);
 
-            this.executeRequest(linkElement.dataset.url, linkElement.dataset.redirectUrl);
+            if(linkElement.dataset.markPageAsRead){
```

Comment:
It's cosmetic, can you please format your code like this:

```
if (true) {
  // ...
} else {
  // ...
}
```

Reply:
Done in 1851f2fb4ca14f522f4df160d24bd6c66d06e683